packages/medusa/src/services/cart.ts

```diff
@@ -1056,9 +1055,7 @@ class CartService extends TransactionBaseService {
   protected async createOrFetchUserFromEmail_(
     email: string
   ): Promise<Customer> {
-    const schema = Validator.string()
-      .email()
-      .required()
+    const schema = Validator.string().email().required()
```

Comment:
suggestion: We want to incrementally move away from `joi` and only use `class-validator`, so I believe we are better off using `validateEmail` from our utils. Goes for the entire file. Let me know what you think

Reply:
yeah sure, it was not part of that PR scoped but I can still do it if you think it should be included in that pr

Reply:
Fine with omitting it for now. I'll create an issue for it instead 👍

Reply:
👍 noted